src/frontend/next/src/components/Post/Post.tsx

```diff
@@ -0,0 +1,203 @@
+import { useRef, useState } from 'react';
+
+import useSWR from 'swr';
+import 'highlight.js/styles/github.css';
```

Comment:
next.js won't let us do this, we'll have to load this globally, see https://nextjs.org/docs/basic-features/built-in-css-support.